packages/styles/tag.css

```
@@ -15,18 +18,23 @@
 .Tag {
   color: var(--tag-text-color);
   background-color: var(--tag-background-color);
-  font-size: var(--text-size-smaller);
   border: 1px solid var(--tag-border-color);
-  border-radius: 11px;
+  border-radius: var(--tag-height);
   display: inline-flex;
   justify-content: center;
   align-items: center;
-  padding: 2px 8px;
+  padding: 0px var(--space-smallest);
```

Nit: `0` values should be unitless in css.

```suggestion
  padding: 0 var(--space-smallest);
```